roles/bettermarks_proxy/templates/apps.conf.j2

```diff
@@ -3,6 +3,11 @@ server{
   location / {
     proxy_hide_header 'Access-Control-Allow-Origin';
     add_header 'Access-Control-Allow-Origin' $http_origin;
+    # Hide original CSP Headers
+    proxy_hide_header 'Content-Security-Policy';
+    proxy_hide_header 'Content-Security-Policy-Report-Only';
+    # We don't know how to add root domain.
+    add_header 'Content-Security-Policy-Report-Only' "default-src 'self' 'unsafe-eval' 'unsafe-inline' *.{{ bettermarks_proxy_maindomain }} {{ root_domain }}; report-uri https://{{ bettermarks_proxy_subdomains['csp'] }}.{{ bettermarks_proxy_maindomain }}/csp/report-only";
```

Comment:
Intent:
Here, we want to add the following config:
Staging:
```
add_header 'Content-Security-Policy-Report-Only' "default-src 'self' 'unsafe-eval' 'unsafe-inline' *.bettermarks.staging.brandenburg.dbildungscloud.org *.staging.brandenburg.dbildungscloud.org; report-uri https://csp-report.bettermarks.staging.brandenburg.dbildungscloud.org/csp/report-only";
```


Production:
```
add_header 'Content-Security-Policy-Report-Only' "default-src 'self' 'unsafe-eval' 'unsafe-inline' *.bettermarks.brandenburg.cloud *.brandenburg.cloud; report-uri https://csp-report.brandenburg.cloud/csp/report-only";
```